plugins/woocommerce/includes/class-woocommerce.php

```diff
@@ -230,6 +230,18 @@ public function legacy_rest_api_is_available() {
 		return class_exists( 'WC_Legacy_REST_API_Plugin', false );
 	}
 
+	/**
+	 * Get the WooCommerce version.
+	 *
+	 * @since 10.2.0
+	 *
+	 * @param bool $include_prerelease_info Whether to include pre-release information in the version string.
+	 * @return string The WooCommerce version.
+	 */
+	public function version( ?bool $include_prerelease_info = false ): string {
+		return $include_prerelease_info ? $this->version : preg_replace( '/-.*/', '', $this->version );
```

Comment:
In this case, we can avoid using regexes (which are relatively heavy): `explode( '-', $this->version, 2 )[0]` would be faster alternative. Here are some examples of how it would handle versions: https://3v4l.org/2WoAk

Reply:
I'm ok with this. Thanks for the suggestion. I don't expect `->version()` to be called a lot during normal operations, but anything counts. I'll make the change.